src/core/include/units/bits/basic_concepts.h

```diff
@@ -102,7 +102,11 @@ template<typename T>
 concept AliasUnit = requires(T* t) { detail::to_base_alias_unit(t); };
 
 // BaseDimension
-template<basic_fixed_string Symbol, NamedUnit U>
+#ifdef __cpp_explicit_this_parameter__
```

Comment:
All `__cpp_explicit_this_parameter__` should lose the trailing `__`.

Reply:
Thanks! I was on my way to Italy yesterday and have Internet only on my cell phone. I retyped it in the wrong way.
Fixed now.